### ISSUE TITLE
Receiving CGR_SESSION_DISCONNECT with multiple kamailio evapi connections

### ISSUE THREAD

**Issue by mintos5** [cgrates/cgrates#1717]
### Description

When we have multiple Kamailio agents in cgrates.json we are receiving **CGR_SESSION_DISCONNECT**, CGRateS is allways terminating the rating call. With just one Kamailio agent in the cgrates.json it is working as it should. 
We discussed this problem in forum: https://groups.google.com/forum/#!topic/cgrates/BhG9a8m0CKo

#### Reproduction
Add multiple EVAPI connections to cgrates.json and expect that all Kamailio sends you correct list of calls

#### Log Messages from Kamailio

```
Oct 23 11:27:23 kamailio-server /opt/ipgate/kamailio/sbin/kamailio[23193]: INFO: <script>: XLOG: 09a7ad63-0f42-4e7c-97ca-e5a2506dddee [CREDIT_REQUEST] Message sent with Body:
{"event":"CGR_AUTH_REQUEST",
"tr_index":"11620",
"tr_label":"2079572401",
"cgr_subsystems":"*attributes;*accounts;",
"reply_route":"CGR_AUTH_REPLY",
"Usage":"7200",
"OriginID":"09a7ad63-0f42-4e7c-97ca-e5a2506dddee;0UDtU97rvNt7a",
"OriginHost":"192.168.6.14:8448",
"RequestType":"*prepaid",
"Tenant":"1518536340",
"Account":"default",
"Subject":"custom",
"Category":"cat_customer_from",
"Destination":"number",
"Origination":"number",
"Customer_from_branch":"branch",
"Customer_to_branch":"branch",
"call_owner":"14567",
"billing_scenario_dialplan_12":"default",
"Origination_date":"2019-10-23",
"Destination_time":"2019-10-23T05:27:23-04:00",
"Destination_date":"2019-10-23",
"SetupTime":"2019-10-23T11:27:23+02:00"}
Oct 23 11:27:23 kamailio-server /opt/ipgate/kamailio/sbin/kamailio[23206]: INFO: <script>: XLOG: 09a7ad63-0f42-4e7c-97ca-e5a2506dddee [CREDIT_AUTHORIZATION] User 1518536340:default can call to number for 336s
Oct 23 11:27:24 kamailio-server /opt/ipgate/kamailio/sbin/kamailio[23206]: INFO: <script>: XLOG: 123 [CGR_DLG_LIST] Requested dialog list
Oct 23 11:27:25 kamailio-server /opt/ipgate/kamailio/sbin/kamailio[23194]: INFO: <script>: XLOG: 09a7ad63-0f42-4e7c-97ca-e5a2506dddee [CGR_CALL_START] Message sent with Body:
{"event":"CGR_CALL_START",
"h_entry":"3711",
"h_id":"1553",
"cgr_subsystems":"*attributes;*accounts",
"OriginID":"09a7ad63-0f42-4e7c-97ca-e5a2506dddee;0UDtU97rvNt7a",
"OriginHost":"192.168.6.14:8448",
"RequestType":"*prepaid",
"Tenant":"1518536340",
"Account":"default",
"Subject":"custom",
"Category":"cat_customer_from",
"Destination":"number",
"Origination":"number",
"Customer_from_branch":"branch",
"Customer_to_branch":"branch",
"call_owner":"14567",
"billing_scenario_dialplan_12":"default",
"Origination_date":"2019-10-23",
"Destination_time":"2019-10-23T05:27:23-04:00",
"Destination_date":"2019-10-23",
"SetupTime":"2019-10-23T11:27:23+02:00",
"AnswerTime":"2019-10-23T11:27:25+02:00"}
Oct 23 11:27:25 kamailio-server /opt/ipgate/kamailio/sbin/kamailio[23194]: INFO: <script>: XLOG: 09a7ad63-0f42-4e7c-97ca-e5a2506dddee [dialog:start] Dialog started

Oct 23 11:27:29 kamailio-server /opt/ipgate/kamailio/sbin/kamailio[23206]: INFO: <script>: XLOG: 123 [CGR_DLG_LIST] Requested dialog list
Oct 23 11:27:29 kamailio-server /opt/ipgate/kamailio/sbin/kamailio[23206]: INFO: <script>: XLOG: 123 [CGR_SESSION_DISCONNECT] Call "3711":"1553" disconnected by reason "INSUFFICIENT_CREDIT" 
```

### Possible Solutions

I found the problem in agents/kamagent.go and fixed it like this in function V1GetActiveSessionIDs:

```
func (ka *KamailioAgent) V1GetActiveSessionIDs(ignParam string, sessionIDs *[]*sessions.SessionID) (err error) {
        for _, evapi := range ka.conns {
                kamEv, _ := json.Marshal(map[string]string{utils.Event: CGR_DLG_LIST})
                if err = evapi.Send(string(kamEv)); err != nil {
                        utils.Logger.Err(fmt.Sprintf("<%s> failed sending event, error %s",
                                utils.KamailioAgent, err.Error()))
                        return
                }
                select {
                case sIDs := <-ka.activeSessionIDs:
                        *sessionIDs = append(*sessionIDs, sIDs...)
                case <-time.After(1 * time.Second):
                        err = errors.New("timeout executing dialog list")
                }
        }
        return
}
```
In short I moved the select to for loop, so all evapi connections can append to sessionIDs their sessions.

### Additional Information

```
 CGRateS 0.9.1~rc8 git+4c92a3c
```
